test/snippet/index/bi_fm_index_iterator.cpp

```diff
@@ -0,0 +1,92 @@
+#include <vector>
+#include <iostream>
+#include <seqan3/index/all.hpp>
+
+using namespace seqan3;
+using namespace seqan3::literal;
+
+int main()
+{
+    {
```

Comment:
in general, don't indent top-level(s) in the snippets, because it will also indent in the documentation.